singer_sdk/sinks/core.py

```diff
@@ -52,6 +50,12 @@ class Sink(metaclass=abc.ABCMeta):
 
     MAX_SIZE_DEFAULT = 10000
 
+    record_validator = fastjsonschema
+    """Default JSONSschema validation library, Developers can override."""
```

Comment:
This doesn't look as great.

<img width="794" alt="Screenshot 2023-12-01 at 1 48 04 p m" src="https://github.com/meltano/sdk/assets/16805946/ed506101-b0d5-4634-8890-9fa70ce01890">

Either we shouldn't document it, or we should create a wrapper that implements certain interface (e.g. RecordValidator) but it's not too useful to document it as is.

Wdyt, the interface thing is probably out of scope?

Reply:
@edgarrmondragon I added add an interface.  Please take a look and let me know what you think.